plugins.txt

```diff
@@ -79,7 +79,7 @@ groovy:457.v99900cb_85593
 handy-uri-templates-2-api:2.1.8-22.v77d5b_75e6953
 htmlpublisher:1.32
 inline-pipeline:1.0.3
-instance-identity:173.va_37c494ec4e5
+instance-identity:185.v303dc7c645f9
```

Comment:
https://plugins.jenkins.io/instance-identity/releases/#version_185.v303dc7c645f9 => JDK21 compliance and a new feature